templates/base.html

```diff
@@ -65,11 +65,25 @@
 
 		<footer class="footer">
 			<div class="container">
-				<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'img/CodeCatz.jpg' %}">
-					<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
-				</a>
-			</div>
+				<div id="footrow">
+					<div class="col-md-4">
+						<ul class="ulfooter">
+							<li class="lifooter"><a href="{% url 'pages-about' %}">Sponsors</a></li>
```

Comment:
By the way, there's really no need to add a class to each `<li>` item. This might help: http://www.w3schools.com/Css/css_combinators.asp